orc8r/gateway/python/magma/magmad/upgrade/upgrader.py

```diff
@@ -13,6 +13,8 @@
 import asyncio
 import logging
 
+DEFAULT_VERSION = '0.0.0-0'
```

Comment:
Any reason, we set a dummy version and not `None`?

Reply:
Good Q, can we figure out where this value is coming from?

Reply:
It's the default in the mconfig and I thought it should be treated the same way as a missing value.

(The MagmaUpgrader has a version check that will always skip if that value is given, I guess that's why it was introduced.)

Reply:
Yeah that makes sense to me! Is the 0 value harcoded somewhere on the Orc8r side? Or is it a default value that protobuf assigns when nothing is specified?